relay-general/src/store/transactions/processor.rs

```diff
@@ -461,23 +472,34 @@ fn scrub_span_description(span: &mut Span) -> Result<(), ProcessingAction> {
     }
 
     let mut scrubbed = span.description.clone();
+    let mut did_scrub = false;
 
     if let Some(is_url_like) = span.op.value().map(|op| op.starts_with("http")) {
         if is_url_like && scrub_identifiers(&mut scrubbed)? {
-            let Some(new_desc) = scrubbed.value() else {
-                    return Ok(());
-                };
-            span.data
-                .get_or_insert_with(BTreeMap::new)
-                // We don't care what the cause of scrubbing was, since we assume
-                // that after scrubbing the value is sanitized.
-                .insert(
-                    "description.scrubbed".to_owned(),
-                    Annotated::new(Value::String(new_desc.to_owned())),
-                );
+            did_scrub = true;
         }
     }
 
+    if let Some(is_db_like) = span.op.value().map(|op| op.starts_with("db")) {
+        if is_db_like && scrub_sql_queries(&mut scrubbed)? {
+            did_scrub = true;
+        }
+    }
+
+    if did_scrub {
+        let Some(new_desc) = scrubbed.value() else {
+            return Ok(());
+        };
+        span.data
+            .get_or_insert_with(BTreeMap::new)
+            // We don't care what the cause of scrubbing was, since we assume
+            // that after scrubbing the value is sanitized.
+            .insert(
+                "description.scrubbed".to_owned(),
+                Annotated::new(Value::String(new_desc.to_owned())),
+            );
```

Comment:
I already left this comment in https://github.com/getsentry/relay/pull/2095#discussion_r1188215948, to repeat: 

* the `scrubbed` var is already cloned description
* you could use `.into_value()` here instead of the `.value()`
* since it would give the the `Option<String>` and there won't be any need to do `new_desc.to_owned()` where you allocate another string again on line `499`